### PR TITLE
verifyMiddleware fails due to bad check in redis-session

### PR DESCRIPTION
**Pull request by daviddahl**

### CODE REVIEW

**Comment by daviddahl** [comment]
@grempe can you review this change? That else if and not checking of the redis call returned anything was causing verifySession to just fail:(

**Comment by daviddahl** [comment]
@fx5 @chiiph if either of you would like to look at this change I would be most appreciative as well!

File: server/redis-session.js [hunk 1]
```diff
@@ -182,20 +182,28 @@ module.exports = function(config) {
...
+              logger.info('err...........', err);
+              if (err || !info) {
+                  logger.error('----get ERROR');
+                  logger.error(err);
+                  info = err;
+                  err  = true;
+                  if (!info) {
+                      logger.error('Session ID invalid!');
+                  }
+                  return callback && callback.apply(mod, [null, true, err]);
```
Comment (fx5) [inline server/redis-session.js]: I believe, you wanted this to be
```
return callback && callback.apply(mod, [null, true, info]);
```
or
```
return callback && callback.apply(mod, [null, err, info]);
```

Comment (daviddahl) [inline server/redis-session.js]: do you mean "instead of"?

Comment (fx5) [inline server/redis-session.js]: no, instead of
```
return callback && callback.apply(mod, [null, true, err]);
```

**Pull request merged**